pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,792 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Onboarding new cluster\n",
+    "This tutorial explains how you can onboard to a new cluster\n",
+    "\n",
+    "## Operation\n",
```

Comment:
This heading was supposed to be distinct between tutorial where user prepares PR themselves, and part where they ask us (through issue) for onboarding their cluster. But since the second part isn't currently in this tutorial I removed it and might consider better wording...